### PR TITLE
Update PR template and little cosmetic in docs

### PR DESCRIPTION
**Pull request by abhishekmishragithub**
## Description
DEV-718

What functionality does this add/problem does this fix?
- Add "DEV" in the ClickUp task URL.
- Add emoji for the type of change
- Little cosmetic changes to coding standard doc page.

This is a 🐁  change.

## Checklist
- [x] Set an informative, carefully chosen **title**
- [x] Applied *labels* to the PR
- [x] Updated README.md and inline documentation as appropriate
- [x] Followed [Coding standards](/docs/coding-standard.md#coding-standard)
- [ ] Added TODO comments about the work not shipped in this PR
- [ ] Documented bugs/issues you had to work around

### CODE REVIEW

**Comment by tchklovski** [comment]
Task linked: [DEV-718 update pull-request-template to 0.4.1](https://app.clickup.com/t/8640607/DEV-718)

**Comment by tchklovski** [comment]
@abhishekmishragithub could you make this 0.4.2 and pull in changes from eg https://raw.githubusercontent.com/predictionmachine/event-tools/main/.github/pull_request_template.md
merging with this?

File: docs/coding-standard.md [hunk 1]
```diff
@@ -8,18 +8,18 @@ please review the basics:
...
+* Do use separate [virtual env](https://docs.python.org/3/library/venv.html) for each repo.
 * IDE:
   * For long-term contributors we prefer that you use the
   [PyCharm](https://www.jetbrains.com/pycharm/) IDE (if you are already
   using IntelliJ, IntelliJ with Python plugin may work)
   * [VSCode](https://code.visualstudio.com/) may also be viable if that's your strong preference
 * Pair Programming:
-  * (beta) We are evaluating [Code With Me](https://plugins.jetbrains.com/plugin/14896-code-with-me)
-as a way to pair on code, a surprisingly fruitful technique
+  * We are evaluating [Duckly](https://duckly.com/tools/pycharm) as a way to pair on code,
```
Comment (tchklovski) [inline docs/coding-standard.md:18]: let's "ship it"
```suggestion
  * We use [Duckly](https://duckly.com/tools/pycharm) as a way to pair on code, 
```

File: docs/coding-standard.md [hunk 2]
```diff
@@ -51,8 +52,8 @@ as a way to pair on code, a surprisingly fruitful technique
 
 ## Github: branches, pull requests
 
-* 🛑 No new repos: do not create a new repository on your own -- request one from tchklovski.
-  This keeps us more organized and naming consistent. In your request, specify what the code
+* 🛑 No new repos: do not create a new repository on your own -- Create a task on [ClickUp](https://app.clickup.com/login) to request one.
```
Comment (tchklovski) [inline docs/coding-standard.md:55]:
```suggestion
* 🛑 No new repos: do not create a new repository on your own -- Create a task on [ClickUp](https://app.clickup.com/login), assign to @manasRK to request one.
```

**Review by tchklovski — APPROVED** [review]
nice, thanks! see comment about merging another update to template

**Comment by abhishekmishragithub** [comment]
@tchklovski please merge if this looks good 🐥

**Review by tchklovski — APPROVED** [review]

**Pull request merged**